tests/Integration/NotifierTest.php

```diff
@@ -105,23 +106,49 @@ public function test_sendAlertsPerEmailToRecipient()
 
     public function test_sendAlertsPerEmailToRecipient_shouldUseDifferentSubjectDependingOnPeriod()
     {
-        $this->assertDateInSubject('week', 'week December 21 - 27, 2009');
-        $this->assertDateInSubject('day', 'Thursday, December 31, 2009');
-        $this->assertDateInSubject('month', 'December 2009');
+        $this->assertDateInSubject('week', $this->getMockMail('week')->getSubject());
```

Comment:
@mattab This is @sebastianpiskorski's PR. I think he might do this because of this: https://github.com/piwik/piwik/blob/2491e6bfcf61984b0f0f3387e1496815e809c6cc/core/Mail.php#L138